cmd/ical-relay/api.go

```diff
@@ -187,7 +195,7 @@ func NotifyRecipientApiHandler(w http.ResponseWriter, r *http.Request) {
 
 	switch r.Method {
 	case http.MethodPost:
-		err := dataStore.AddNotifyRecipient(notifier, mail)
+		err := dataStore.AddNotifyRecipient(notifier, datastore.Recipient{Recipient: mail, Type: "email"})
```

Comment:
we should be able to create every type of recipient here

Reply:
Das hab ich mir auch schon gedacht. Aktuell gibt die API das nicht her...

Reply:
magst du das implementieren mit einem Fallback zu email um die API nicht zu breaken & ggf kurz testen? Meine Docker Umgebung macht immernoch probleme wodurch testing schwierig ist...

Reply:
der webhook typ wäre für discord wahrscheinlich auch sehr interessant

Reply:
Ja, da ist halt aber die Frage wie man das implementiert. Gibts da ein Standard Webhook Format? Bei Discord kann man ja auch noch eine Menge Styling machen

Reply:
Ich glaube es gibt kein Default webhook format. Das ist halt einfach HTTP POST. Üblicherweise gibt der Poster das Format vor. Aber wir können natürlich uns an Discord erstmal anpassen.

Reply:
https://discord.com/developers/docs/resources/webhook#execute-webhook

Reply:
Meine Überlegung wäre ansonsten, dass wir ein generelles Format machen und ein Discord-spezifisches Format, wenn in der URL discord.com enthalten ist